app/src/main/res/layout/activity_recycler_view.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/activity_recycler_view"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="#466"
+    tools:context="com.sdsmdg.hareshkh.lectureassignment.arihant.Activities.RecyclerViewActivity">
+    <android.support.v7.widget.RecyclerView
```

Comment:
RecyclerView is an extension of a viewgroup, and can be used as root layout in absence of other views, which is the case here.